synthetic_signwriting/synthetic_signwriting.py

```diff
@@ -1,37 +1,86 @@
 import random
 
+import numpy as np
 from pose_format import Pose
 
+from synthetic_signwriting.hands.hands import read_npy_file, prorate_hand
+from synthetic_signwriting.faces.faces import prorate_face, read_pkl_file
+from synthetic_signwriting.synthetic_pose_helper import randomize_hand_pose, create_base_frame, pose_transition
+
 
 class SyntheticSignWriting:
-    pose = Pose(header=None, body=None)
 
     def __init__(self):
-        self.key_frames = []
+        with open('data/base_pose.pose', 'rb') as file:
+            data_buffer = file.read()
+        pose = Pose.read(data_buffer)
+        middle = int(len(pose.body.data) / 2)
+
+        self.hand_data = read_npy_file('data/hands.npy')
+        self.face_data = read_pkl_file('data/faces.pkl')
+        self.pose = pose
+        self.static_position = [pose.body.data[middle + 2 * i][0].copy() for i in range(4)]
         self.hands = []
```

Comment:
possibly consolidate these two:
`self.sequence = []`

then used a `namedtuple` with `face, left_hand, right_hand` and push that (to not deal with two arrays), and to make it extendable with the `face_signwriting`, etc